linux-object/src/fs/fcntl.rs

```diff
@@ -0,0 +1,39 @@
+//! consts for fctnl
```

Comment:
note: some of the consts in this file should be converted into `enum` or `bitflags` in the future.